### ISSUE TITLE
The `description` schema field is not used in the Admin UI

### ISSUE THREAD

**Issue by JMLX42** [keystonejs/keystone#7325]
## How to reproduce

1. Define [the `description` field of a schema](https://keystonejs.com/docs/apis/schema#description).
2. Open the Admin UI on that schema.

## Expected behavior

The expected behavior is [the documented one](https://keystonejs.com/docs/apis/schema#description):

> The description option defines a string which will be used as a description in the Admin UI

## Actual behavior

The schema description is not in the Admin UI.

```
"@keystone-6/auth": "^1.0.1",
"@keystone-6/core": "^1.0.1",
"@keystone-6/fields-document": "^1.0.1",
```

**Comment by pkuwyn** [comment]
I asked the same in slack;
the keystone team said:"This feature isn’t fully operational yet ,We’ll look into this next sprint."

**Comment by dcousens** [comment]
Relevant discussion for fields (not yet implemented) can be found at https://github.com/keystonejs/keystone/discussions/5875

**Comment by dcousens** [comment]
Fixed in #7537
